app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2 class="login-heading">Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "username" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password", placeholder: "password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field" id="remem">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ Required ] I think it would be better if the design of your `registration` and `login` forms appear exactly as they are in the project wireframes for this milestone. As a developer, you'll be required to bring out designs just as they appear in design documents. Kindly see the screenshots below and rectify this.

## Project wireframe
<img src="https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png" width="375px" />

## Your design 
<img src="https://user-images.githubusercontent.com/16392046/231541577-897822f6-24a1-4800-b590-81f02dc17422.png" width="375px" />

Reply:
Modified the design ✅